modules/database/src/ioc/db/dbConvertFast.h

```diff
@@ -19,8 +19,11 @@
 extern "C" {
 #endif
 
-DBCORE_API extern long (*dbFastGetConvertRoutine[DBF_DEVICE+1][DBR_ENUM+1])();
-DBCORE_API extern long (*dbFastPutConvertRoutine[DBR_ENUM+1][DBF_DEVICE+1])();
+struct dbAddr;
+typedef long (*FASTCONVERTFUNC)(const void *from, void *to, const struct dbAddr *paddr);
```

Comment:
Core Group: Check build of external modules which are using this, if any. Search for the `dbFast{Get,Put}ConvertRoutine()` functions.

Reply:
Looking in the epics-modules organization only `dbFastPutConvertRoutine` is found [here in autosave](https://github.com/epics-modules/autosave/blob/606903e177790c6431b277d4393700d9e5991b26/asApp/src/dbrestore.c#L80) and that's in a comment referring to code [added in 2004](https://github.com/epics-modules/autosave/commit/636baf20d6596d5cb84592b2ac04b04be052ae02) but [removed in 2017](https://github.com/epics-modules/autosave/commit/3a58565341d895f33d28ae0386697f2295542a82). Forks of autosave provide many references to this old code in a GitHub-wide search.

@klemenv's [PyDevice module](https://github.com/klemenv/PyDevice) is also using `dbFastPutConvertRoutine` from C++ code [here](https://github.com/klemenv/PyDevice/blob/ec1ef37a51f2ca3183c42baafa25c482e4822dc7/src/pycalcRecord.cpp#L210). I'm not sure if this change will break that code or not, but it looks like fixing it shouldn't be too hard with a version check for backwards compatibility.

Other uses of both dbFast{Get,Put}ConvertRoutine on GitHub all appear to be forks of Base or of the above.